debugger/Cargo.toml

```diff
@@ -37,7 +41,12 @@ serde = { version = "^1.0", features = ["derive"] }
 schemafy = "^0.6"
 chrono = { version = "0.4", features = ["serde"] }
 goblin = "0.4.1"
-defmt-decoder = { version = "0.2.1", features = ["unstable"] }
+gimli = { version = "0.26", default-features = false, features = [
+    "endian-reader",
+    "read",
+    "std",
+] }
```

Comment:
I think I fell victim to this default setting `rust-analyzer.completion.autoimport.enable (default: true)` ... if you choose the wrong autocompletion, then it adds an import. I probably backspaced on the incorrect code, instead of doing an `undo`.

I have removed the dependency and it will show up in the next commit.